src/storage/src/hummock/sstable/mod.rs

```diff
@@ -311,6 +362,10 @@ impl SstableMeta {
         for tombstone in &self.range_tombstone_list {
             tombstone.encode(buf);
         }
+        buf.put_u32_le(self.monotonic_tombstone_events.len() as u32);
+        for monotonic_tombstone_event in &self.monotonic_tombstone_events {
+            monotonic_tombstone_event.encode(buf);
```

Comment:
> Why do we need to encode `monotonic_tombstone_event`? It can be reconstructed by `range_tombstone_list`.

This is introduced in the (almost) latest commit 1324231b . We can revert the commit.